library/std/src/keyword_docs.rs

```diff
@@ -1448,6 +1448,9 @@ mod self_upper_keyword {}
 /// in a multithreaded context. As such, all accesses to mutable `static`s
 /// require an [`unsafe`] block.
 ///
+/// Instead of `static mut`, it's safer to use a regular `static`, and interior
+/// mutability types such as [`Mutex`], [`OnceLock`], or [atomic] types.
```

Comment:
```suggestion
/// When possible, it's often better to use a non-mutable `static` with an
/// interior mutable type such as [`Mutex`], [`OnceLock`], or an [atomic].
```

The logic behind the suggestion is to flow better with the following paragraph and to sidestep a bit the question of what is "safer".  E.g., though not listed, `SyncUnsafeCell` is also an interior mutable type, but saying whether or not that is safer is a bit nuanced.